hyperledger/grid
# Add docker dev and test environments for Grid UI

- Adds unit tests and lint to Jenkins
- Adds a sapling-dev-server directory which can be used for testing Grid saplings
- Adds a new docker-compose file for testing Grid UI
- Removes and updates some boilerplate in Grid UI

To test in Docker:

from the Grid repo, enable experimental features and run `docker-compose -f grid-ui/docker/docker-compose.yaml up`. The Grid UI is available at localhost:3005.

To test locally:

From `grid/grid-ui`, run `npm install && yarn start`. The Grid UI should be available at localhost:3000. 

Navigate to different saplings and ensure that there are no errors.
